statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -884,7 +887,11 @@ def _get_robustcov_results(self, cov_type='opg', **kwargs):
         res.cov_kwds = {}
 
         # Calculate the new covariance matrix
-        if self.cov_type == 'cs':
+        if len(self.params) == 0:
```

Comment:
What's the case if params is empty? 
(just a question, I never checked the behavior of any model in that case, if other empty models are even possible)